README.md

```diff
@@ -149,24 +151,84 @@ loadConfig[Conf](conf)
 ```
 
 
-## Customizing naming conventions
+## Extend the library to support new types
+
+Not all types are supported automatically by `pureconfig`. For instance, classes
+that are not case classes are not supported out-of-the-box:
+
+```scala
+> class Foo(var bar: Int) { override def toString: String = s"Foo(${bar.toString})" }
+> loadConfig[Foo]
+could not find implicit value for parameter conv: pureconfig.ConfigConvert[Foo]
```

Comment:
Nitpick: This line should be visually distinct in some way to indicate that it is a compiler error message rather than straight up code. Typically we do that with a leading `//`